batavia/builtins/compile.js

```diff
@@ -2,6 +2,11 @@ var exceptions = require('../core').exceptions
 
 function compile(args, kwargs) {
     var _compile = require('../modules/_compile/_compile')
+    
+    if (args.length < 3) {
+        var argument_names = ['source', 'filename', 'mode']
+        throw new exceptions.TypeError.$pyclass("Required argument '" + argument_names[args.length] + "' (pos " + (args.length + 1) + ") not found")
+    }
 
```

Comment:
At column 1: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)